src/main/java/org/apache/commons/imaging/formats/gif/GifImageParser.java

```diff
@@ -1003,7 +1003,6 @@ public void writeImage(final BufferedImage src, final OutputStream os, Map<Strin
                 // boolean LocalColorTableFlag = false;
                 final boolean interlaceFlag = false;
                 final boolean sortFlag = false;
-                final int sizeOfLocalColorTable = colorTableScaleLessOne;
```

Comment:
I think the author here intentionally made a redundant variable so other developers would understand the code more easily. Imaging has a lot of code with many variables, or methods implementing papers and using the variables as they appear in the papers.

So any help like this helps, so IMO we should keep issues like that when found by static analyzers / linters.

Reply:
HI @kinow 
Changed
TY